Cannot deploy to OSSRH Project description missing, Project URL missing, Developer information missing

FO] [ERROR] Repository "orgbkvm-1003" failures
[INFO] [ERROR]   Rule "pom-staging" failures
[INFO] [ERROR]     * Invalid POM: /org/bkvm/bookkeeper-visual-manager-parent/1.1.0/bookkeeper-visual-manager-parent-1.1.0.pom: Project description missing, Project URL missing, Developer information missing
[INFO] [ERROR] 
[INFO] [ERROR] 
[INFO] [ERR

--- a/pom.xml
+++ b/pom.xml
@@ -5,9 +5,35 @@
     <packaging>pom</packaging>
     <groupId>org.bkvm</groupId>
     <version>1.1.0-SNAPSHOT</version>
-
     <name>BookKeeper Visual Manager - Parent</name>
-
+    <url>https://bkvm.org</url>
+    <developers>
+        <developer>
+            <id>mino181295</id>
+            <name>Matteo Minardi</name>
+            <organization>Diennea</organization>
+        </developer>
+        <developer>
+            <id>eolivelli</id>
+            <name>Enrico Olivelli</name>
+            <organization>Diennea</organization>
+        </developer>
+        <developer>
+            <id>alessandro.luccaroni</id>
+            <name>Alessandro Luccaroni</name>
+            <organization>Diennea</organization>
+        </developer>
+        <developer>
+            <id>dcdiennea</id>
+            <name>Diana clementi</name>
+            <organization>Diennea</organization>
+        </developer>
+        <developer>
+            <id>piercarlobeghetto</id>
+            <name>Piercarlo Beghetto</name>
+            <organization>Diennea</organization>
+        </developer>
+    </developers>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
@@ -227,6 +253,14 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-javadoc-plugin</artifactId>
+                    <version>3.2.0</version>
+                    <configuration>
+                        <detectJavaApiLink>false</detectJavaApiLink>
+                    </configuration>
+                </plugin>
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-war-plugin</artifactId>

--- a/pom.xml
+++ b/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>bookkeeper-visual-manager-parent</artifactId>
+    <artifactId>bkvm-parent</artifactId>
     <packaging>pom</packaging>
     <groupId>org.bkvm</groupId>
     <version>1.1.0-SNAPSHOT</version>

--- a/site-skin/pom.xml
+++ b/site-skin/pom.xml
@@ -20,7 +20,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <groupId>org.bkvm</groupId>
-        <artifactId>bookkeeper-visual-manager-parent</artifactId>
+        <artifactId>bkvm-parent</artifactId>
         <version>1.1.0-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>

--- a/site/pom.xml
+++ b/site/pom.xml
@@ -20,7 +20,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <groupId>org.bkvm</groupId>
-        <artifactId>bookkeeper-visual-manager-parent</artifactId>
+        <artifactId>bkvm-parent</artifactId>
         <version>1.1.0-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>

--- a/standalone/pom.xml
+++ b/standalone/pom.xml
@@ -2,7 +2,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <groupId>org.bkvm</groupId>
-        <artifactId>bookkeeper-visual-manager-parent</artifactId>
+        <artifactId>bkvm-parent</artifactId>
         <version>1.1.0-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>
@@ -19,7 +19,7 @@
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
-            <artifactId>bookkeeper-visual-manager</artifactId>
+            <artifactId>bkvm</artifactId>
             <version>${project.version}</version>
             <type>war</type>
         </dependency>
@@ -69,7 +69,7 @@
                 <version>3.1.1</version>
                 <configuration>
                     <appendAssemblyId>false</appendAssemblyId>
-                    <finalName>bookkeeper-visual-manager-${project.version}</finalName>
+                    <finalName>bkvm-${project.version}</finalName>
                     <descriptors>
                         <descriptor>src/main/assemble/zip-assembly.xml</descriptor>
                     </descriptors>

--- a/standalone/src/main/assemble/zip-assembly.xml
+++ b/standalone/src/main/assemble/zip-assembly.xml
@@ -30,13 +30,13 @@
             <useStrictFiltering>true</useStrictFiltering>
             <unpack>true</unpack>           
             <includes>                
-                <include>org.bkvm:bookkeeper-visual-manager:war</include>
+                <include>org.bkvm:bkvm:war</include>
             </includes>            
             <outputDirectory>web</outputDirectory>
         </dependencySet>
         <dependencySet>
             <excludes>
-                <exclude>org.bkvm:bookkeeper-visual-manager:war</exclude>
+                <exclude>org.bkvm:bkvm:war</exclude>
             </excludes>
             <useProjectArtifact>true</useProjectArtifact>
             <useStrictFiltering>true</useStrictFiltering>

--- a/web/pom.xml
+++ b/web/pom.xml
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>bookkeeper-visual-manager</artifactId>
+    <artifactId>bkvm</artifactId>
     <packaging>war</packaging>
     <parent>
         <groupId>org.bkvm</groupId>
-        <artifactId>bookkeeper-visual-manager-parent</artifactId>
+        <artifactId>bkvm-parent</artifactId>
         <version>1.1.0-SNAPSHOT</version>
         <relativePath>..</relativePath>
     </parent>

